gnssanalysis/gn_io/sp3.py

```diff
@@ -25,10 +25,25 @@
                                     (\w+)(?:[ ]+(\w+)|)""",
     _re.VERBOSE,
 )
-# SV names. multiline, findall
+# Regex to extract Satellite Vehicle (SV) names (E.g. G02). Regex options/flags: multiline, findall
 _RE_SP3_HEAD_SV = _re.compile(rb"^\+[ ]+(?:[\d]+|)[ ]+((?:[A-Z]\d{2})+)\W", _re.MULTILINE)
-# orbits accuracy codes
-_RE_SP3_ACC = _re.compile(rb"^\+{2}[ ]+([\d\s]{50}\d)\W", _re.MULTILINE)
+
+# Regex for orbit accuracy codes (E.g. ' 15' - space padded, blocks are three chars wide).
+# Note: header is padded with '  0' entries after the actual data, so empty fields are matched and then trimmed.
+_RE_SP3_HEADER_ACC = _re.compile(rb"^\+{2}[ ]+((?:[\-\d\s]{2}\d){17})$", _re.MULTILINE)
```

Comment:
It turns out the use of the trailing `$` here was a mistake. Not all SP3 files have the line ending immediately after the orbit accuracy codes. Some pad with spaces. This is resolved in https://github.com/GeoscienceAustralia/gnssanalysis/pull/45